low-level/framework/utils/conf_utils.py

```diff
@@ -167,3 +170,35 @@
 BMC_IP_KEY = "server_node>%s>bmc>ip" % MACHINE_ID
 BMC_USER_KEY = "server_node>%s>bmc>user" % MACHINE_ID
 BMC_SECRET_KEY = "server_node>%s>bmc>secret" % MACHINE_ID
+
+conf_dict = {
+    "site_id": SITE_ID_KEY + "__default:" + DEFAULT_DC,
+    "node_id": NODE_ID_KEY + "__default:" + DEFAULT_SN,
+    "cluster_id": CLUSTER_ID_KEY + "__default:" + DEFAULT_CLUSTER,
+    "rack_id": RACK_ID_KEY + "__default:" + DEFAULT_RACK,
+    "cortx_build": CORTX_BUILD_KEY,
+    "storage_type": STORAGE_TYPE_KEY + "__default:virtual",
+    "server_type": NODE_TYPE_KEY + "__default:virtual"
+}
+
+def fetch_config(conf_list, index='GLOBAL'):
+    try:
+        response = {}
+        default_val = ""
+        for key in conf_list:
+            if key not in conf_dict:
+                raise KeyError(f"key:{key} not found in the config.")
+            output = conf_dict[key]
+            if "__default" in output:
+                req_string = output.split("__")[0]
+                temp_val = output.split("__")[1]
```

Comment:
`suggestion` 
can be done in single line : `req_string, temp_val = output.split("__")`

Reply:
@sumedhak27 `split` returns a single list of splited elements, so multiple assignment may not work here.

Reply:
Yes. This is a better way.

Reply:
Done.